.github/workflows/release.yml

```diff
@@ -82,31 +83,36 @@ jobs:
             echo "Creating tag: $TAG_NAME"
             if git tag -a "$TAG_NAME" -m "Release $MODULE version $VERSION" && git push origin "$TAG_NAME"; then
               echo "Tagged $MODULE with version $VERSION"
-              
-              # Add to the JSON array of created tags
-              CREATED_TAGS_JSON=$(echo $CREATED_TAGS_JSON | jq --arg tag "$TAG_NAME" '. += [$tag]')
+              CREATED_TAGS+=("$TAG_NAME")
             else
               echo "Error creating or pushing tag $TAG_NAME"
             fi
           done
 
-          # Check if we created any tags
-          TAGS_COUNT=$(echo $CREATED_TAGS_JSON | jq 'length')
-          TAGS_CREATED=false
-          if [ "$TAGS_COUNT" -gt 0 ]; then
-            TAGS_CREATED=true
-          fi
-
-          # Set output variables for next steps
-          echo "tags_created=$TAGS_CREATED" >> $GITHUB_OUTPUT
-          echo "created_tags_json=$CREATED_TAGS_JSON" >> $GITHUB_OUTPUT
-
-          if [ "$TAGS_CREATED" = "true" ]; then
-            echo "Successfully created $TAGS_COUNT tag(s)"
+          # Convert array to JSON and set outputs
+          if [ ${#CREATED_TAGS[@]} -gt 0 ]; then
+            CREATED_TAGS_JSON=$(printf '%s\n' "${CREATED_TAGS[@]}" | jq -R . | jq -s .)
```

Comment:
Robots helped me understand this but I'm wondering if there are long form versions of these flags for JQ that would allow a reader to more easily understand what's being done here?

Reply:
I think so. That's a good idea.